.github/workflows/cancel_redundant_workflows.yml

```diff
@@ -0,0 +1,20 @@
+name: Cancel redundant workflows
+on:
+  workflow_run:
+    types:
+      - requested
+    workflows:
+      - clang-format
+      - Lint
+      - Test tools
+
+jobs:
+  cancel:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Cancel duplicate workflow runs
+        uses: potiuk/cancel-workflow-runs@v4_8
```

Comment:
just making sure--this is the version we used yesterday right?

Reply:
yes: yesterday we were using `@master`, which (as of 2021-02-09) points to https://github.com/potiuk/cancel-workflow-runs/commit/a81b3c4d59c61e27484cfacdc13897dd908419c9, the same commit as [v4_8](https://github.com/potiuk/cancel-workflow-runs/releases/tag/v4_8).